keras_cv/layers/preprocessing/auto_contrast.py

```diff
@@ -64,6 +64,9 @@ def augment_image(self, image, transformation=None, **kwargs):
         result = tf.where(tf.math.is_nan(result), original_image, result)
         return result
 
+    def augment_bounding_boxes(self, bounding_boxes, transformation, **kwargs):
```

Comment:
drop transformation, it isn't used

Reply:
I tried to keep it consistent the BaseImageAugmentationLayer call(). I'll update it.